services/NaturalLanguageUnderstanding/index.md

```diff
@@ -0,0 +1,100 @@
+---
+
+copyright:
+  years: 2017
+lastupdated: "2017-03-24"
+
+---
+
+{:new_window: target="_blank"}
+{:shortdesc: .shortdesc}
+{:screen: .screen}
+{:codeblock: .codeblock}
+{:pre: .pre}
+
+# Getting started with Natural Language Understanding
+{: #natural-language-understanding}
+
+With {{site.data.keyword.nlushort}}, developers can analyze semantic features of unstructured text to extract structured insights. The service returns information for categories, concepts, emotion, entities, keywords, metadata, relations, semantic roles, and sentiment.
+{:shortdesc}
+
+Follow these steps to get started with {{site.data.keyword.nlushort}}.
+
+## Create a service instance
+
+Go to the [Natural Language Understanding](https://bluemix.net/catalog/services/natural-language-understanding) page in the service catalog, log in to Bluemix, and click **Create**.
```

Comment:
I missed this in my PR to staging - adding the domain name var. Also, doesn't the url include `console.`? `https://`**console.{DomainName}**`/catalog/services/natural-language-understanding`